linkml_model/model/schema/meta.yaml

```diff
@@ -2410,3 +2534,33 @@ enums:
         meaning: skos:broaderMatch
       NARROW_SYNONYM:
         meaning: skos:narrowerMatch
+
+  # I'm not really sure if this is valid/correct, but I wanted to give a quick and dirty example. This is based on: https://build.fhir.org/valueset-example-intensional.json.html - joeflack4 2022/05/27
+  serum_cholesterol_enum:
+    # `description`: I didn't see this as a valid `slot` for `enum_expression`. Should this be here?
+    description: >
+      This is an example value set that includes all the LOINC codes for serum/plasma cholesterol from v2.36.
+    # `is_a`: This too. is this / should this be here?
+    is_a: enum_expression
+    code_set: http://hl7.org/fhir/ValueSet/serum-cholesterol
+    # code_set_tag: development
+    code_set_version: "1.0.0"
+    pv_formula: CODE
+    include:
+      - LP43571-6
+    minus:
+      - 5932-9
+    # inherits:
+    reachable_from:
+      source_ontology: http://loinc.org
+      source_nodes: LP43571-6  # not sure if I did this right... there's only one include code, defined as: {"property": "parent", "op": "=", "value": "LP43571-6"}
```

Comment:
`source_nodes`: not sure if I did this right... there's only one include code, defined in the FHIR JSON example as: `{"property": "parent", "op": "=", "value": "LP43571-6"}`